rust-runtime/inlineable/src/event_receiver.rs

```diff
@@ -18,6 +18,14 @@ impl<T, E> EventReceiver<T, E> {
         Self { inner }
     }
 
+    #[doc(hidden)]
+    // Wrapper around `try_recv_initial` on `aws_smithy_http::event_stream::Receiver`
+    //
+    // Note: This method is intended for internal use only.
+    pub async fn try_recv_initial(&mut self) -> Result<Option<Message>, SdkError<E, RawMessage>> {
```

Comment:
Because this is an inlineable can't we just make this `pub(crate)`?

Reply:
Fixed in 2ffa396.